xbmc/cores/RetroPlayer/process/gbm/RenderBufferGBM.h

```diff
@@ -0,0 +1,74 @@
+/*
+ *      Copyright (C) 2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include "cores/RetroPlayer/buffers/BaseRenderBuffer.h"
+#include "utils/GBMBufferObject.h"
```

Comment:
Can be forward-delcared

Reply:
Move definition to .cpp file. IMO only reason to have definition in header if it's a single variable or returning a container's .size() value. IMO multiplication qualifies the definition for the .cpp (because you might want to set a breakpoint to check each value), and sacrificing a forward-declare definitely does.